internal/app/linker.go

```diff
@@ -0,0 +1,30 @@
+package linker
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/goware/urlx"
+	"github.com/kisanetik/learn_go_inc1/config"
+)
+
+func CompressURL(url string) string {
+	tFile, err := os.CreateTemp("", "")
+	if err != nil {
+		panic(err)
+	}
+	os.WriteFile(tFile.Name(), []byte(url), 0644)
+
+	return string(makeHostFromConfig() + "/" + filepath.Base(tFile.Name()))
+}
+
+func makeHostFromConfig() string {
+	_, port := config.LoadConfig()
+
+	url, _ := urlx.Parse(config.GetConf().BaseURL)
+	host, port1, _ := urlx.SplitHostPort(url)
+	if port1 != "" {
+		return url.Scheme + "://" + host + ":" + port1
```

Comment:
Please don't use `+`
P.S. https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go

Reply:
ok, I was use fmt.sprintf, in that case maybe better for easy reading instead string.writestring or join from map at your examples